### PR TITLE
Improve documentation: Add `How to access the mailkick table`

### PR DESCRIPTION
**Pull request by manojmj92**
I am adding this to include information that could have benefited me, but was found missing in the documentation.

I had to mass update the existing list names and since `mailkick_opt_outs` is not exposed as a model, I thought the only way to do this was via raw SQL.

But turns out that simply opening a class with the same name which inherits from `ActiveRecord::Base` works and now mass updates, searching data for debugging etc are much more easier.

I have added this information. Hope it benefits somebody :)

UPDATE: Mailkick already exposes data via `Mailkick::OptOut`, updated docs to use this class

### CODE REVIEW

**Comment by ankane** [comment]
Hey @manojmj92, thanks for the PR 👍 There is a model, but it's named `Mailkick::OptOut`, so maybe we can make a note of that in the docs.

**Comment by ankane** [comment]
Hey @manojmj92, feel like it's a bit wordy right now. I think we can add a sentence to the reference section about how you can access opt outs directly with `Mailkick::OptOut`.

**Comment by manojmj92** [comment]
@ankane How about now?

**Comment by ankane** [comment]
Sorry for the delay - thanks @manojmj92 👍
